lib/core_extensions/array.rb

```diff
@@ -2,7 +2,7 @@ class Array
   def entropy
     each_with_object(Hash.new(0)) do |i, result|
       result[i] += 1
-    end.values.sum do |count|
+    end.values.inject(0, :+) do |count|
```

Comment:
`sum` is clearer than `inject(0, :+)`. However, `sum` without a block will not work. This is a valid fix.

Reply:
I think it was because `sum` didn't exist outside of Rails / `activesupport`, but in my environment I had those and so I didn't notice this.

Reply:
Hmm, it is part of the Array: https://ruby-doc.org/core-2.4.0/Array.html#method-i-sum

Reply:
True, in 2.4.0 ;)

https://ruby-doc.org/core-2.3.0/Array.html

Reply:
Fair enough :)